sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -719,8 +719,7 @@ case class GpuLike(left: Expression, right: Expression, escapeChar: Char)
     } else {
       null
     }
-    val regexStr = escapeLikeRegex(likeStr, escapeChar)
```

Comment:
We should remove escapeLikeRegex entirely. The new code appears to be better in every way.